temporian/implementation/numpy/operators/drop_index.py

```diff
@@ -0,0 +1,181 @@
+from collections import defaultdict
+from typing import Dict, List, Union
+
+import numpy as np
+
+from temporian.core.operators.drop_index import DropIndexOperator
+from temporian.implementation.numpy.data.event import NumpyEvent
+from temporian.implementation.numpy.data.event import NumpyFeature
+from temporian.implementation.numpy.data.sampling import NumpySampling
+
+IndexMetadata = Dict[
+    str, Union[int, List[List[NumpyFeature]], List[np.ndarray]]
+]
+
+
+def _impl(
```

Comment:
agree ☝🏼 and in this case I would even argue that having this logic inside the class is fine, since it's the only thing the class is actually implementing